doc/netplan.md

```diff
@@ -242,8 +242,33 @@ Virtual devices
     configured and DHCP is disabled, the interface may still be brought online,
     but will not be addressable from the network.
 
+    In addition to the addresses themselves one can specify configuration
+    parameters as mappings. Current supported options are:
+
+    ``lifetime`` (scalar)
+    :    Default: ``forever``. This can be ``forever``, ``infinity`` or ``0`` and
+         corresponds to the ``PreferredLifetime`` option in
+         ``systemd-networkd``'s Address section. Currently only has an effect on
+         the ``networkd`` backend.
+
+    ``label`` (scalar)
+    :    An address label. Currently only has an effect on the ``networkd``
+         backend.
```

Comment:
there are two different uses of 'address label' in ip:
 - ip address label
 - ip addrlabel

I think it should be clearly documented here which this refers to (i.e. the first one).

Reply:
I've tried detailing this a bit more in the docs. Please let me know if we still want more details on this!